afl-cmin.py

```diff
@@ -312,6 +329,22 @@ def main():
             already_have.add(t)
         count += 1
 
+    if args.crash_dir:
```

Comment:
Before your patch, crashes will be stored in `output_path`.
But after your patch, crashes will be skip if `crash_dir` is not specified.
Could you preserve the previous behavior?

Reply:
Are you sure about this? I think this is only true if `AFL_CMIN_ALLOW_ANY=1` is set since otherwise `afl-showmap` doesn't return coverage for a crash so they are never included in the minimized set.

In any case, I've made afl-cmin.py have the same behaviour as before if `--crash-dir` is not passed. I did a quick test to ensure it works, namely:

1. `afl-cmin.py -i in -o out ...` doesn't retain crashes in `out` and doesn't save them anywhere.

   It also doesn't print how many crashes it encountered. I'm not too sure about this. Do we want it to always print the number of crashes?

2. `AFL_CMIN_ALLOW_ANY=1 afl-cmin.py -i in -o out ...` considers crashes as regular inputs and retains them if they happen to be effective (have novel tuples and less in size than all other files that have those tuples).
3. `afl-cmin.py -i in -o out --crash-dir crash ...` separates out crashes into the `crash` dir and prints how many it found when printing the number of effective files.

Anything else?